java/metric-scaler/src/main/java/com/example/bigtable/scaler/MetricScaler.java

```diff
@@ -144,6 +144,7 @@ public MetricScaler(String projectId, String instanceId)
    * @throws IOException
    */
   Point getLatestValue() throws IOException {
+    // [START bigtable_cpu]
     // [START get_bigtable_cpu]
```

Comment:
should this be deleted?

Reply:
It should once the docs are updated — this is used on c.g.c. so I need to replace the region tags there first.